FirebaseRemoteConfigSwift/Sources/PropertyWrapper/RemoteConfigValueObservable.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseRemoteConfig
+import FirebaseCore
+import SwiftUI
+
+extension Notification.Name {
+  // Listens to FirebaseRemoteConfig SDK if new configs are activated.
+  static let onRemoteConfigActivated = Notification.Name("FIRRemoteConfigActivateNotification")
+}
+
+// Make sure this key is consistent with kFIRGoogleAppIDKey in FirebaseCore SDK
+let FirebaseRemoteConfigGoogleAppIDKey = "FIRGoogleAppIDKey"
```

Comment:
Could we just make the FirebaseCore constant public?

Reply:
Will do as we make this notification public. This might need a different API review.